src/index.js

```diff
@@ -5,13 +5,35 @@ if (!TESTING)
 
 const { EventEmitter } = require('events');
 
+function appendQueryParameter (url, name, value) {
+  if (!url.includes("?")) {
+    url += "?"
+  } else {
+    url += "&"
+  }
+  const urlEscaper = new URLSearchParams()
+  urlEscaper.set(name, value)
+  url += urlEscaper.toString()
+  return url
+}
+function makeUrlAbsolute (url) {
+  const link = document.createElement("a");
+  link.href = url;
+  return (link.protocol + "//" + link.host + link.pathname)
+}
+
 class Connection extends EventEmitter {
 
-  constructor(url, signalCb) {
+  constructor(url, signalCb, branding) {
     super();
 
     const hostname = window.location.hostname;
     this.chaperone_url = url || `http://${hostname}:24273`;
+    if (branding !== undefined) {
+      this.chaperone_url = appendQueryParameter(this.chaperone_url, "logo_url", makeUrlAbsolute(branding.logo_url))
+      this.chaperone_url = appendQueryParameter(this.chaperone_url, "app_name", branding.app_name)
+    }
```

Comment:
```suggestion
   if (!!branding.logo_url) {
      this.chaperone_url = appendQueryParameter(this.chaperone_url, "logo_url", makeUrlAbsolute(branding.logo_url))
    }
    if (!!branding.app_name) {
      this.chaperone_url = appendQueryParameter(this.chaperone_url, "app_name", branding.app_name)
    }
```

Reply:
Let's separate these so that it's easier to filter out nullish values in chaperone.  Currently, if the logo_url is undefined or null that value will be appended to the localhost domain string, and deceptively inform chaperone that the logo_url does exist.

Reply:
Good catch. I believe as written this would cause as error like `cannot access property logo_url of undefined` if branding was `undefined`. I'll commit a slight modification.